client/src/components/LAButton.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="button-wrapper">
-    <div class="button" :style="{ width: width }">
+    <div class="button" :style="{ width: width, margin: margin }">
       <p class="answer-number" v-if="number">{{ number }}</p>
```

Comment:
Don't have to do this now, but eventually we can make this a [slot](https://vuejs.org/guide/components/slots.html) so that we can add arbitrary content in here (like images). If you'd like to before merging, we could include the google and facebook icons on the buttons themselves by doing this.

Reply:
Included the icons using a slot.

Reply:
Even better, since we have the slot, we also wouldn't need number and text as parameters anymore. I can go ahead and remove these in a follow up though.